Percolator/Percolator.tla

```diff
@@ -25,30 +24,39 @@ VARIABLES client_ts
 \* prewrite.
 VARIABLES client_key
 
-\* $key_data[k]$ is the set of multi-version data timestamp of the key.
-\* Only start_ts.
+\* $key_data[k]$ is the set of multi-version data of the key.
+\* Since we don't care about the concrete value of data, a record of
+\* [ts: start_ts] is sufficient to represent one data version.
 VARIABLES key_data
 
-\* $key_lock[k]$ is the set of lock. A lock is of a record
-\* [ts: ts, primary: lock]. $ts$ is for start_ts. If $primary$ equals to $k$,
-\* it is a primary lock, otherwise secondary lock.
+\* $key_lock[k]$ is the set of lock. A lock is of a record of
+\* [ts: start_ts, primary: lock]. If $primary$ equals to $k$, it is a primary
+\* lock, otherwise secondary lock.
 VARIABLES key_lock
 
 \* $key_write[k]$ is a sequence of committed version of the key.
-\* A committed version of the key is a record of [start_ts: ts, commit_ts: ts].
+\* A committed version of the key is a record of [ts: commit_ts,
```

Comment:
I prefer using commit_ts and start_ts.

Reply:
Isn't good that `key_data[k]`, `key_lock[k]` and `key_write[k]` all have a `ts` field to emphasize that they are indexed by timestamp?

Another reason is, we are going to add `rollback` in `key_write`, but the indexing `ts` for `rollback` is `start_ts`, while it is `commit_ts` for normal `write`. I believe unifying everything with `ts` is better than the old data organization. There should be no confusion if we comment it clear enough.